.gitignore

```diff
@@ -1 +1,4 @@
 ._*
```

Comment:
I don't think these file extensions need to be ignored as the code repository should not be cloned into where a user's script resides.  Rather the encpass.sh script right now should be copied out of the folder of the cloned repo and into the folder where the script resides.  Eventually the installation instructions will be changed to have the user add encpass.sh to the user's path as encpass.sh will support holding the keys in a directory (.encpass) under the user's home directory.  Also, on the off chance I need to include a file with these extensions I'd rather not add them to the .gitignore.